server/config/config.go

```diff
@@ -0,0 +1,35 @@
+// Package config contains logic and defaults for the otrego server.
+//
+// Relies on https://github.com/kelseyhightower/envconfig, which is inspired by
+// 12-factor app configuration: https://12factor.net/confi.
+package config
+
+import (
+	"log"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+const (
+	// envVarPrefix is a prefix to all environment variables.  For example, to
+	// specify FOO, you will actually need to specify FOO, unless the struct tag
+	// `envconfig:"OVERRIDE"` is supplied
+	envVarPrefix = "OTREGO"
+)
+
+// Spec is the dynamic configuration spec for the server, which is modified
```

Comment:
Did you mean to add more to this sentence after "which is modified"?

Reply:
Oops, thanks.